kmpauth-google/src/androidMain/kotlin/com/mmk/kmpauth/google/GoogleAuthUiProviderImpl.kt

```diff
@@ -25,24 +25,43 @@ internal class GoogleAuthUiProviderImpl(
         return try {
             val credential = credentialManager.getCredential(
                 context = activityContext,
-                request = getCredentialRequest()
+                request = getCredentialRequest(checkPreviouslyUsedAccounts = true)
```

Comment:
For now it is okay just passing `true` here. But when I make new version with these changes, I'll add this varaible to configuration and by default will set it to `false` as before. Because I think there is no downside setting false except in some cases it causes crash which  I think that bug will be handled by Google.

Reply:
Thanks for consideration. I agree!